olclient/cli.py

```diff
@@ -88,6 +88,16 @@ def main():
         config_tool.update(config)
         return "Successfully configured "
 
+    # prompt first time users to configure credentials for olclient
+    try:
+        if not len(ia.config.get_config()):
```

Comment:
This checks whether the ia tool configuration exists, which is different to the OL client's own config.

The config is accessible via `Config.get_config()` https://github.com/internetarchive/openlibrary-client/blob/master/olclient/config.py#L125

but for the purposes of this feature you should simply be able to wrap line 101 `ol = OpenLibrary()` in the try and except on the `ValueError`

Reply:
> but for the purposes of this feature you should simply be able to wrap line 101 `ol = OpenLibrary()` in the try and except on the `ValueError`

This seems like a much simpler solution. I shall make the suggested changes.
Thank you @hornc.